container/dockershell.sh

```diff
@@ -171,17 +181,23 @@ fi
 mkdir -p ${USER_VOLUME}
 # start container in interactive mode
 
-entryPoint="env && ${SETUP_PATH}/setup-ssl.sh && ${SETUP_PATH}/entrypoint_msg.sh && bash --login"
+echo "setting the entrypoint variable"
+echo $SETUP_PATH
```

Comment:
What's the point of printing this, I would just delete it.

Reply:
those were added by me to help with debugging, have removed them